handler/src/main/java/io/netty/handler/ssl/SslContext.java

```diff
@@ -766,7 +768,9 @@ static ApplicationProtocolConfig toApplicationProtocolConfig(Iterable<String> ne
     /**
      * Creates a new instance.
      */
-    protected SslContext() { }
```

Comment:
we still need a default constructor to preserve the API ... just have it call `this(false);`